spec/unit/openapi-renderer/endpoint/toPathObject.spec.ts

```diff
@@ -1139,7 +1221,7 @@ describe('OpenapiEndpointRenderer', () => {
               context('requestBody leverages for opt', () => {
                 it('uses the model provided in the for option to determine request body shape', () => {
                   const renderer = new OpenapiEndpointRenderer(Pet, ApiPetsController, 'create', {
-                    requestBody: { for: User, only: ['email'], required: ['email'] },
+                    requestBody: { for: User, only: ['email'], required: ['id'], including: ['id'] },
```

Comment:
This is odd because it is nested within a "requestBody is not specified" context.